scripts/apps/authoring/authoring/directives/SendItem.js

```diff
@@ -176,9 +181,9 @@ export function SendItem($q, api, desks, notify, authoringWorkspace,
                 }
             };
 
-            scope.send = function(open) {
+            scope.send = function(open, allPackageItems) {
```

Comment:
Rename `allPackageItems` to `moveAllPackageItems` or `sendAllPackageItems`. Unless there's a verb in front, the name, it implies that the argument is not a bool, but a collection of items.